? calendar/gui/apps_evolution_calendar-2.2.schemas
? calendar/gui/dialogs/.calendar-setup.c.swp
? help/C/evolution-2.2-C.omf
? help/C/evolution-2.2.xml
? mail/evolution-mail-2.2.schemas
Index: calendar/gui/migration.c
===================================================================
RCS file: /cvs/gnome/evolution/calendar/gui/migration.c,v
retrieving revision 1.31
diff -u -r1.31 migration.c
--- calendar/gui/migration.c	28 Sep 2004 15:57:33 -0000	1.31
+++ calendar/gui/migration.c	30 Sep 2004 11:02:07 -0000
@@ -366,6 +366,7 @@
 #define WEBCAL_BASE_URI "webcal://"
 #define CONTACTS_BASE_URI "contacts://"
 #define BAD_CONTACTS_BASE_URI "contact://"
+#define WEATHER_BASE_URI "weather://"
 #define PERSONAL_RELATIVE_URI "system"
 
 static ESourceGroup *
@@ -394,7 +395,8 @@
 			 ESourceGroup **on_this_computer,
 			 ESource **personal_source,
 			 ESourceGroup **on_the_web,
-			 ESourceGroup **contacts)
+			 ESourceGroup **contacts,
+			 ESourceGroup **weather)
 {
 	GSList *groups;
 	ESourceGroup *group;
@@ -487,7 +489,14 @@
 
 		*contacts = group;
 	}
-	
+
+	if (!*weather) {
+		group = e_source_group_new (_("Weather"), WEATHER_BASE_URI);
+		e_source_list_add_group (source_list, group, -1);
+
+		*weather = group;
+	}
+
 	g_free (base_uri_proto);
 	g_free (base_uri);
 }
@@ -699,14 +708,14 @@
 gboolean
 migrate_calendars (CalendarComponent *component, int major, int minor, int revision, GError **err)
 {
-	ESourceGroup *on_this_computer = NULL, *on_the_web = NULL, *contacts = NULL;
+	ESourceGroup *on_this_computer = NULL, *on_the_web = NULL, *contacts = NULL, *weather = NULL;
 	ESource *personal_source = NULL;
 	gboolean retval = FALSE;
 
 	/* we call this unconditionally now - create_groups either
 	   creates the groups/sources or it finds the necessary
 	   groups/sources. */
-	create_calendar_sources (component, calendar_component_peek_source_list (component), &on_this_computer, &personal_source, &on_the_web, &contacts);
+	create_calendar_sources (component, calendar_component_peek_source_list (component), &on_this_computer, &personal_source, &on_the_web, &contacts, &weather);
 
 	if (major == 1) {
 		xmlDocPtr config_doc = NULL;
@@ -842,6 +851,8 @@
 		g_object_unref (on_the_web);
 	if (contacts)
 		g_object_unref (contacts);
+	if (weather)
+		g_object_unref (weather);
 	if (personal_source)
 		g_object_unref (personal_source);
 
Index: calendar/gui/dialogs/calendar-setup.c
===================================================================
RCS file: /cvs/gnome/evolution/calendar/gui/dialogs/calendar-setup.c,v
retrieving revision 1.29
diff -u -r1.29 calendar-setup.c
--- calendar/gui/dialogs/calendar-setup.c	30 Jul 2004 07:37:02 -0000	1.29
+++ calendar/gui/dialogs/calendar-setup.c	30 Sep 2004 11:02:07 -0000
@@ -187,12 +187,13 @@
 	 * silently translate an explicit file:// into http:// though. */
 	if (uri->protocol &&
 	    strcmp (uri->protocol, "http") &&
-	    strcmp (uri->protocol, "webcal")) {
+	    strcmp (uri->protocol, "webcal") &&
+	    strcmp (uri->protocol, "weather")) {
 		e_uri_free (uri);
 
 		if (interactive)
 			e_notice (parent, GTK_MESSAGE_ERROR,
-				  _("The source location '%s' is not a webcal source."),
+				  _("The source location '%s' is not a weather or webcal source."),
 				  source_location);
 		return FALSE;
 	}
Index: mail/em-filter-i18n.h
===================================================================
RCS file: /cvs/gnome/evolution/mail/em-filter-i18n.h,v
retrieving revision 1.1
diff -u -r1.1 em-filter-i18n.h
--- mail/em-filter-i18n.h	17 Jun 2004 07:34:49 -0000	1.1
+++ mail/em-filter-i18n.h	30 Sep 2004 11:02:09 -0000
@@ -4,36 +4,19 @@
 char *s = N_("Assign Score");
 char *s = N_("Attachments");
 char *s = N_("Beep");
-char *s = N_("contains");
 char *s = N_("Copy to Folder");
 char *s = N_("Date received");
 char *s = N_("Date sent");
 char *s = N_("Delete");
 char *s = N_("Deleted");
-char *s = N_("does not contain");
-char *s = N_("does not end with");
-char *s = N_("does not exist");
-char *s = N_("does not return");
-char *s = N_("does not sound like");
-char *s = N_("does not start with");
 char *s = N_("Do Not Exist");
 char *s = N_("Draft");
-char *s = N_("ends with");
 char *s = N_("Exist");
-char *s = N_("exists");
 char *s = N_("Expression");
 char *s = N_("Follow Up");
 char *s = N_("Important");
-char *s = N_("is");
-char *s = N_("is after");
-char *s = N_("is before");
-char *s = N_("is Flagged");
-char *s = N_("is greater than");
-char *s = N_("is less than");
-char *s = N_("is not");
-char *s = N_("is not Flagged");
-char *s = N_("Junk");
 char *s = N_("Junk Test");
+char *s = N_("Junk");
 char *s = N_("Label");
 char *s = N_("Mailing list");
 char *s = N_("Message Body");
@@ -47,19 +30,36 @@
 char *s = N_("Recipients");
 char *s = N_("Regex Match");
 char *s = N_("Replied to");
-char *s = N_("returns");
-char *s = N_("returns greater than");
-char *s = N_("returns less than");
 char *s = N_("Run Program");
 char *s = N_("Score");
 char *s = N_("Sender");
 char *s = N_("Set Status");
 char *s = N_("Size (kB)");
-char *s = N_("sounds like");
 char *s = N_("Source Account");
 char *s = N_("Specific header");
-char *s = N_("starts with");
 char *s = N_("Status");
 char *s = N_("Stop Processing");
 char *s = N_("Subject");
 char *s = N_("Unset Status");
+char *s = N_("contains");
+char *s = N_("does not contain");
+char *s = N_("does not end with");
+char *s = N_("does not exist");
+char *s = N_("does not return");
+char *s = N_("does not sound like");
+char *s = N_("does not start with");
+char *s = N_("ends with");
+char *s = N_("exists");
+char *s = N_("is Flagged");
+char *s = N_("is after");
+char *s = N_("is before");
+char *s = N_("is greater than");
+char *s = N_("is less than");
+char *s = N_("is not Flagged");
+char *s = N_("is not");
+char *s = N_("is");
+char *s = N_("returns greater than");
+char *s = N_("returns less than");
+char *s = N_("returns");
+char *s = N_("sounds like");
+char *s = N_("starts with");
